bedrock/stories/templates/stories/articles/art-of-engagement.html

```diff
@@ -0,0 +1,148 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "stories/stories-base-article.html" %}
+
+{% from "stories/includes/macros.html" import stories_article_blockquote, stories_image with context %}
+
+
+{% block publish_date %}2022-08-02{% endblock %}
+{% block publish_date_formatted %}August, 2, 2022{% endblock %}
+
+
+
+
+{% set featured = true %}
+{% set credits = [
+  {"name": "Ray Fembro", "title": "Product Manager", "image": "img/stories/credit/ray-fembro-credit.png"},
+  {"name": "Ania Safko", "title": "Product Manager", "image": "img/stories/credit/ania-safko-credit.png"},
+  {"name": "Jon Siddoway", "title": "Community Manager", "image": "img/stories/credit/jon-siddoway-credit.png"},
+] %}
+
+{% set related = [
+   {
+    "title": "Licensed to Build Together",
+    "desc": "Who really owns the Firefox browser? Its copyleft license means we all do.",
+    "image_url": "img/stories/build-together.png",
+    "link_url": url('stories.articles.build-together'),
+    "image_alt": "Someone placing wooden blocks together"
+  },
+  {
+    "title": "From Volunteer to Mozilla Community Champion",
+    "desc": "An early Mozilla volunteer's curiosity about open-source software led him to a career with a company whose mission he believes in.",
```

Comment:
Nit: curly apostrophe.
```suggestion
    "desc": "An early Mozilla volunteer’s curiosity about open-source software led him to a career with a company whose mission he believes in.",
```